server/src/internalClusterTest/java/org/elasticsearch/discovery/DiscoveryDisruptionIT.java

```diff
@@ -229,7 +229,7 @@ public void testJoinWaitsForClusterApplier() throws Exception {
             } catch (Exception e) {
                 throw new AssertionError(e);
             }
-        }, ActionListener.wrap(() -> {}));
+        }, ActionListener.running(() -> {}));
```

Comment:
Do we have many of `ActionListener.wrap(() -> {})`? May be it is worth replacing them with `ActionListener.noop()`?

Reply:
This is the only one I see that should be a `noop()`.